src/apimtemplate/Core/ArmTemplates/ResourceCreators/ApiResourceCreator.cs

```diff
@@ -187,7 +187,7 @@ private IEnumerable<ArmTemplateResource<ApiProperties>> CreateApis(DeploymentDef
 
 			static string GetApiName(ApiDeploymentDefinition apiDeploymentDefinition)
 			{
-				if (int.TryParse(apiDeploymentDefinition.ApiRevision, out var revisionNumber) && revisionNumber >= 1 && apiDeploymentDefinition.IsCurrent == false)
```

Comment:
Thanks. I believe `IsCurrent  != true` would be simpler to understand. Thoughts?

Reply:
Ah yes, that's definitely simpler, haha my bad, will fix :)

Reply:
Fixed